shared/oae/api/oae.api.util.js

```diff
@@ -1346,7 +1346,7 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
          * Redirect the currently user to the me page.
          */
         var me = function() {
```

Comment:
I've changed it, but I can't actually find anywhere in the code base that was using `redirect.me()` so maybe the entire method isn't necessary.